readyapi4j-groovy-dsl/src/main/groovy/com/smartbear/readyapi4j/dsl/pro/ProDslDelegate.groovy

```diff
@@ -0,0 +1,29 @@
+package com.smartbear.readyapi4j.dsl.pro
+
+import com.smartbear.readyapi4j.dsl.DslDelegate
+
+/**
+ * The delegate responding to commands inside the "recipe" closure of ServerTestDsl.
+ */
+class ProDslDelegate extends DslDelegate {
```

Comment:
ok thanks - so in line with that thinking - should we rename the testserver-client submodule to readyapi4j-pro ?